package.json

```diff
@@ -102,7 +102,6 @@
     "stylelint": "^13.3.2",
     "stylelint-config-standard": "^20.0.0",
     "stylelint-order": "^4.1.0",
-    "terser-webpack-plugin": "^4.1.0",
```

Comment:
are we expecting to see changes in yarn.lock file as well?

Reply:
Yes, we don't need this plugin anymore as it was only used in the terser webpack config.

Reply:
Right, I will update the yarn.lock